analyzer-java/src/main/java/com/infosupport/ldoc/analyzerj/descriptions/EnumMemberDescription.java

```diff
@@ -13,12 +13,8 @@ public record EnumMemberDescription(
     List<Description> arguments,
 
     @JsonProperty("DocumentationComments")
-    @JsonInclude(JsonInclude.Include.NON_EMPTY)
```

Comment:
Maybe NON_NULL would be nicer: this would prevent the json from containing a literal 'null' as well as still displaying an empty value '{ }' when the comment description is set